hadoop-hdds/client/src/main/java/org/apache/hadoop/hdds/scm/OzoneClientConfig.java

```diff
@@ -481,16 +481,12 @@ public void setDataStreamBufferFlushSize(long dataStreamBufferFlushSize) {
     this.dataStreamBufferFlushSize = dataStreamBufferFlushSize;
   }
 
-  public ChecksumCombineMode getChecksumCombineMode() {
-    try {
-      return ChecksumCombineMode.valueOf(checksumCombineMode);
-    } catch (IllegalArgumentException iae) {
-      LOG.warn("Bad checksum combine mode: {}. Using default {}",
-          checksumCombineMode,
-          ChecksumCombineMode.COMPOSITE_CRC.name());
-      return ChecksumCombineMode.valueOf(
-          ChecksumCombineMode.COMPOSITE_CRC.name());
```

Comment:
Why don't we return `null` here instead of duplicating validation logic in callers?

Reply:
Done.